gef.py

```diff
@@ -95,17 +96,21 @@
     # Compat Py2/3 hacks
     def range(*args):
         """Replace range() builtin with an iterator version."""
-        if len(args) < 1:
-            raise TypeError()
-        start, end, step = 0, args[0], 1
-        if len(args) == 2: start, end = args
-        if len(args) == 3: start, end, step = args
-        for n in itertools.count(start=start, step=step):
-            if (step>0 and n >= end) or (step<0 and n<=end): break
+        if len(args) == 0:
```

Comment:
This is really confusing, does it really give a gain in performance?
Also this is unrelated to this PR. If you want to re-write the `range()` function, please make a separate PR.